server/database/queries/cohort/getCohorts.js

```diff
@@ -0,0 +1,5 @@
+const connection = require('../../config/connection');
+
+const getCohorts = () => connection.query('SELECT * FROM cohort');
+
+module.exports = { getCohorts };
```

Comment:
Since you are only exporting one function from this file and there's no intent to create and export any other functions from this file, so you should default export `getCohorts` function instead of export it by name.
and don't forget to edit the first require as shown below.
```suggestion
module.exports = getCohorts;
```

Reply:
ok